miopengemm/CMakeLists.txt

```diff
@@ -70,6 +77,7 @@ rocm_create_package(
     DESCRIPTION "A tool for generating OpenCL matrix multiplication (GEMM) kernels"
     MAINTAINER "Paul Fultz II <paul.fultz@amd.com>"
     LDCONFIG
+    LDCONFIG_DIR "/opt/rocm/lib64"
```

Comment:
The installation is into `/lib` and not `/lib64`. Also wont this break if the `CMAKE_INSTALL_PREFIX` is set to `/usr/local`?

Reply:
I will correct the path to /opt/rocm/lib.
The main idea of adding /opt/rocm/lib into the ld.conf, is to help user who can installed multiple versions of ROCm and have the symlink /opt/rocm pointing to the required version they want to use. This enables the loader to load the corresponding lib from based on the ld.conf entry.

To get this behavior and support installing the package into different location(/usr/local), above changes are not sufficient. Please let me know your opinion on this.